library/ssl_tls.c

```diff
@@ -9988,8 +9988,8 @@ int mbedtls_ssl_get_key_exchange_md_tls1_2( mbedtls_ssl_context *ssl,
     psa_hash_operation_t hash_operation;
     psa_algorithm_t hash_alg = mbedtls_psa_translate_md( md_alg );
 
-    MBEDTLS_SSL_DEBUG_MSG( 2, ( "Perform PSA-based computation of digest \
-    		                     of ServerKeyExchange" ) );
+    MBEDTLS_SSL_DEBUG_MSG( 1, ( "Perform PSA-based computation of digest "
```

Comment:
When it comes to debug messages, we break our rule of keeping lines below 80 characters, for the benefit of people grepping for error messages. Could you please keep the error string in one line?